### DISCUSSION TITLE
In python how to write Azure  DevOps pipeline in Json

### DISCUSSION THREAD

**Discussion by RanjeetDhole** [json-schema-org/community#299] [General]
In python how to write Azure  DevOps pipeline in Json

**Comment by Relequestual** [comment]
Hello. Unfortuantly I think you've come to the wrong place @RanjeetDhole.
This is a discussion forum for the JSON Schema specification. JSON Schema is a standard used to validate JSON data.
It has nothing specifically do do with Azure, devops pipelines or otherwise.
Best of luck finding your answer.
